pkg/cli/admin/release/extract.go

```diff
@@ -133,8 +142,11 @@ func NewExtract(f kcmdutil.Factory, streams genericclioptions.IOStreams) *cobra.
 	flags.StringVar(&o.CommandOperatingSystem, "command-os", o.CommandOperatingSystem, "Override which operating system command is extracted (mac, windows, linux) or can be specified with arch(linux/arm64, mac/amd64). You map specify '*' to extract all tool archives.")
 	flags.StringVar(&o.FileDir, "dir", o.FileDir, "The directory on disk that file:// images will be copied under.")
 
-	flags.BoolVar(&o.CredentialsRequests, "credentials-requests", o.CredentialsRequests, "Extract credential request manifests only")
-	flags.StringVar(&o.Cloud, "cloud", o.Cloud, "Specify the cloud for which credential request manifests should be extracted. Works only in combination with --credentials-requests.")
+	flags.BoolVar(&o.Included, "included", o.Included, "Exclude manifests that are not expected to be included in cluster reconciliation.")
+	flags.StringVar(&o.InstallConfig, "install-config", o.InstallConfig, "Path to an install-config file, as consumed by the openshift-install command.  Works only in combination with --included.")
```

Comment:
The assumption here is that install-config only makes sense for manifests which will be reconciled by CVO which is part of `included`. Can you clarify? I am wondering  why we can not call install-config and included separately even if the result might be same for installed clusters?

Reply:
With `--included` the extraction will have manifests that the CVO is expected to include.  Without `--included`, the extraction will have those and also have manifests that the CVO is expected to exclude.  So we need to know "is the CVO expected to include this manifest?" with `--included`, but we don't care about trying to predict the CVO without `--included`.  And because the install config is one way for us to get the information we need to try to predict the CVO, we don't need an install-config without `--included`.  That was a lot of words, but the backing Go may be more clear:

```go
		if o.Included {
			context := "connected cluster"
			inclusionConfig := manifestInclusionConfiguration{}
			if o.InstallConfig == "" {
				inclusionConfig, err = findClusterIncludeConfig(ctx, o.RESTConfig)
			} else {
				inclusionConfig, err = findClusterIncludeConfigFromInstallConfig(ctx, o.InstallConfig)
				context = o.InstallConfig
			}
```

Folks who set `--install-config` without setting `--included` would just have the install-config ignored, so we are failing fast to point out the unnecessary argument and allow the user to re-evaluate what they're trying to accompllish.